AnbProduct.php

```diff
@@ -316,12 +316,11 @@ function topDealProductsNew($atts, $tabName = "")
             $tabItem .= '<i class="' . $tabIcon . '" /></i> ';
         }
         $tabItem .= '<span class="hidden-xs">' . $tabName . '</span><span class="visible-xs">' . $atts['tabname_short'] . '</span></a></li>';
-
-        //TODO: replace energyTopDeals class
+        
         $script = '<script>
                     jQuery(document).ready(function($){
-                        $(\'.energyTopDeals .filterDeals ul\').append(\'' . $tabItem . '\'); 
-                        $(\'.energyTopDeals .dealsTable .tab-content\').append(\'' . $this->minifyHtml($tabContent) . '\');
+                        $(\'.top-deals .filterDeals ul\').append(\'' . $tabItem . '\'); 
```

Comment:
als je hier nu ook meteen .filterDeals vervangt met .tabs oid en .dealsTable hieronder weghaalt, dan zijn we ook meteen van die niet consistente css classes af